### ISSUE TITLE
Think about what to do in case bookies run out of disk space

### ISSUE THREAD

**Issue by mjaakko** [HSLdevcom/transitdata#122]
Currently when bookies run out of disk space (this happens if there's a bug in the application and it cannot process the messages it receives), the whole transitdata system will crash and become unresponsive. Think about what to do if this happens: should we discard older data which causes data loss?

**Comment by pierre-leonard** [comment]
If the server crashes, won't we lose data anyway?

**Comment by mjaakko** [comment]
Yes. But if we discard messages when the application cannot process them, there is a risk of losing data that could have been processed later. There might not be a good solution for this.

**Comment by mjaakko** [comment]
Pulsar tiered storage could help with this: https://pulsar.apache.org/docs/en/tiered-storage-overview/
